kaskade/main.py

```diff
@@ -79,10 +79,22 @@ def cli() -> None:
         metavar="filename",
     ),
 )
+@cloup.option_group(
+    "Cloud configuration options",
+    cloup.option(
+        "--cloud-config",
+        "cloud_config",
+        help="Cloud configuration properties. Typically used to set 'aws.region', which is required when using AWS MSK IAM authentication. Multiple '--cloud-config' are allowed",
+        metavar="property=value",
+        multiple=True,
+        callback=tuple_properties_to_dict,
+    ),
+)
 def admin(
     bootstrap_servers: str,
     kafka_config_file: str | None,
     kafka_config: dict[str, str],
+    cloud_config: dict[str, str],
```

Comment:
It's also possible to validate the configs and example: https://github.com/sauljabin/kaskade/blob/main/kaskade/main.py#L327

We can list all the aws available configurations https://github.com/sauljabin/kaskade/blob/main/kaskade/configs.py#L6